src/server/ns_turn_server.h

```diff
@@ -40,7 +40,10 @@ extern "C" {
 
 //////////// defines //////////////
 
-#define TURN_SESSION_ID_FACTOR (1000000000000000LL)
+// Even if a single threaded app handles on average about 1 session/s
```

Comment:
i suspect 1 session per second is on the low side. My production instances handle many thousands of concurrent users per thread.

Not that your analysis is wrong in the general sense, just that maybe the explanation should be a bit more cautious.

Reply:
Actually the unit (`k`) is missing (1k * 1G = 1T =~ 12 zeros, 1 Gs =~ 32 years). Even if a box handles 50k session/s, session ID would wrap after ~231 days: unlikely that all sessions last that long ...